mesh/node.c

```diff
@@ -1550,7 +1911,7 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 	} else {
 		/* Callback for create node request */
 		node_ready_func_t cb = req->cb;
-		struct keyring_net_key net_key;
+		net_key = l_new(struct keyring_net_key, 1);
 		uint8_t dev_key[16];
```

Comment:
`mesh/node.c:1915:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]`